import sys
print("Something failed")
sys.exit(1)
